mlir/lib/Dialect/Affine/Analysis/Utils.cpp

```diff
@@ -12,6 +12,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir/Dialect/Affine/Analysis/Utils.h"
+
```

Comment:
Please remove this new line

Reply:
This new line should be there per style - to separate the main header for this file from the group of other includes.